test/blackbox-tests/test-cases/vendor/main.t/run.t

```diff
@@ -15,6 +15,12 @@ When compiling vendored code, all warnings should be disabled
   Entering directory 'warnings'
   There should be no OCaml warning!
 
+When compiling vendored code, all alerts should be disabled as well
+
+  $ dune build --root alerts @no-alerts-please
+  Entering directory 'alerts'
+  There should be no OCaml alerts!
+
```

Comment:
Maybe we should also test that warnings / alerts are indeed shown if the `vendored_dirs` stanza is removed ?